docs/wfc3tools/wf3rej.rst

```diff
@@ -40,16 +40,27 @@ as specified by CRSIGMAS. See `Section 3.4.5 of the WFC3 Data Handbook
 for more information.
 
 
+Running `wf3rej` from a Python Terminal
+=======================================
+
+.. code-block:: shell
+
+    from wfc3tools import wf3rej
+    wf3rej([filename1, filename2])
+
+
 Displaying Output from `wf3rej` in a Jupyter Notebook
-=====================================================
+-----------------------------------------------------
 
 When calling `wf3rej` from a Jupyter notebook, informational text output from the underlying `wf3rej.e` program will be passed through `print` as the calibration runs by default, and show up in the user's cell. This behavior can be customized by passing your own function as the `log_func` keyword argument to `wf3rej`. As output is read from the underlying program, the `wf3rej` Python wrapper will call `log_func` with the contents of each line. Note that `print` is an obvious choice for a log function, but this also provides a way to connect `wf3rej` to the Python logging system by passing the logging.debug function or similar.
 
 If `log_func=None` is passed, informational text output from the underlying program will be ignored, but the program's exit code will still be checked for successful completion.
 
+Input Parameters for the Python Interface 
+-----------------------------------------
 
 Parameters
-==========
+~~~~~~~~~~
 
     input : str or list
         Name of input files, such as
```

Comment:
Will this work as a string? Don't you need two files to run `wf3rej`? If so, can it be read as a string of two files (e.g. `files = 'ipppssoot1.fits ipppssoot2.fits'`)?

Reply:
Yes, it works as comma-separated filenames with no spaces (``iaao01k8q_flc.fits,iaao01k9q_flc.fits``).  I have not only updated the docs, but I updated the docstring in the wf3rej.py file.  I also used better filenames in case someone wants to test.

Addressed in commit 3fb2bf2.